Makefile

```diff
@@ -5,7 +5,7 @@ NAME = conjure-up
 CURRENT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 TOPDIR := $(shell basename `pwd`)
 GIT_REV := $(shell git log --oneline -n1| cut -d" " -f1)
-VERSION := $(shell ./tools/version)
+VERSION := 2.0.0.8
```

Comment:
So the Makefile is the source for determining the version since it does some sed replacing lower in the `release` target